lib/controllers/base.js

```diff
@@ -16,6 +16,7 @@ Controller.prototype.initialize = function(options) {
   this.app = options.app;
   this.resource = options.resource;
   this.odm = options.odm;
+  this.isRestify = options.isRestify || true;
```

Comment:
this expression always evaluates to `true` no matter what you pass in. Instead you should do `!!options.isRestify`?